server/bed.js

```diff
@@ -66,8 +68,10 @@ bed.query.motifs = function(query, bedPath) {
   var chr = query.chr;
   var dir = bedPath + fileName + '_chr/' + fileName + '_chr' + chr;
   if (!fs.existsSync(dir)) {
+    var error = 'bed file ' + fileName + ' not found.';
+    log.serverLog([error]);
```

Comment:
Umm using an array of strings could be counter-intuitive. Consider a function with variable-length argument, such as console.log, genotet.error (front-end).